Framework/API/test/FileFinderTest.h

```diff
@@ -471,7 +470,6 @@ class FileFinderTest : public CxxTest::TestSuite {
     TS_ASSERT(path.find("CSp78173.Raw") != std::string::npos);
 #else
     TS_ASSERT(path.find("CSP78173.raw") != std::string::npos);
-#endif
```

Comment:
I think this the replace script is removing too many #endif macros

Reply:
I thaught i caught that one, the script is still in progress.